src/catchup/VerifyLedgerChainWork.cpp

```diff
@@ -94,46 +95,41 @@ VerifyLedgerChainWork::getStatus() const
     {
         std::string task = "verifying checkpoint";
         return fmtProgress(mApp, task, mRange.first(), mRange.last(),
-                           mCurrCheckpoint);
+                           (mRange.last() - mCurrCheckpoint));
     }
     return Work::getStatus();
 }
 
 void
 VerifyLedgerChainWork::onReset()
 {
-    auto setLedger = mApp.getLedgerManager().getLastClosedLedgerHeader();
-    if (setLedger.header.ledgerSeq < 2)
-    {
-        setLedger = {};
-    }
-    if (mFirstVerified.header.ledgerSeq != 0)
-    {
-        mFirstVerified = setLedger;
```

Comment:
Is it save to not reset mFirstVerified here? I think so, as mFirstVerified is verified at the end of this work, so the work cannot fail after that. If my analysis is correct, we should have a comment here explaining that.